notebooks/ai-ml/HuggingFaceEmbeddingGenerationInSpark.ipynb

```diff
@@ -0,0 +1,959 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    results = predict_batch_udf(predict_batch_fn,
</pre><p>Let's rename this var to generate_embeddings_udf?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudDataproc/cloud-dataproc/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>